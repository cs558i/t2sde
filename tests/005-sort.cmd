#!/bin/sh

[ "$QEMU" ] || exit 43

[ -e $SYSROOT/bin/sort ] || exit 42

printf 'car,(20)\njeep,[10]\ntruck,(5)\nbus,[3]' | $QEMU -chroot $SYSROOT bin/sort -t, -k2.2,2n
