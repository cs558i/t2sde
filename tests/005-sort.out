bus,[3]
truck,(5)
jeep,[10]
car,(20)
